internal/model/docker_info.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 type ImageTarget struct {
-	Ref          container.RefSelector
-	BuildDetails BuildDetails
+	ConfigurationRef container.RefSelector
+	DeploymentRef    container.RefSelector
```

Comment:
i don't think you actually want this to be a Selector? i.e., selector matching is never the right semantic behavior on deployed resources. again, see https://github.com/windmilleng/tilt/pull/1337